book/forms.md

```diff
@@ -360,16 +360,18 @@ sheets[^update-styles] every time you type!
     area definitely can't make such changes, so let's skip this in our
     browser.
     
-Now when we click an `input` element and clear its contents, we can
-call `render` to redraw the page with the `input` cleared:
+Now when we click an `input` element and clear its contents, we should
+call `render` to redraw the page with the `input` cleared. To make things
+easier, I'm just always calling `render` regardless of whether things changed:
 
 ``` {.python}
 class Tab:
     def click(self, x, y):
         while elt:
             elif elt.tag == "input":
                 elt.attributes["value"] = ""
-                return self.render()
+                break
+        self.render()
```

Comment:
How does this work with form submission? I think we'll end up rendering twice, once when loading the new page and once again when returning back to the click handler.

Reply:
Yes, but is that so bad? It's just a bit redundant, right?